startup.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-python3 -m app.consumer
+python3 -m app.main
```

Comment:
Will this require a change to gitlab config?

Reply:
James has already changed this.